### PR TITLE
Making script platform agnostic and more additions to make it Studio compliant

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

File: src/android/libs/SalesforceSDK/build.gradle [hunk 1]
```diff
@@ -2,6 +2,7 @@
 apply plugin: 'android-library'
 
 dependencies {
+  compile 'com.facebook.react:react-native:0.12.+'
```
Comment (bhariharan) [inline src/android/libs/SalesforceSDK/build.gradle]: I also pulled in the latest from the ```Android``` source repo.

File: tools/postinstall-android.js [hunk 2]
```diff
@@ -84,19 +80,21 @@ console.log('Fixing application AndroidManifest.xml');
 fixFile(path.join('platforms', 'android', 'AndroidManifest.xml'), fixAndroidManifest);
 
 console.log('Moving Salesforce libraries to the correct location');
-exec('cp -R ' + path.join(libProjectRoot, 'SalesforceSDK') + ' ' + appProjectRoot);
-exec('cp -R ' + path.join(libProjectRoot, 'SmartStore') + ' ' + appProjectRoot);
-exec('cp -R ' + path.join(libProjectRoot, 'SmartSync') + ' ' + appProjectRoot);
+shelljs.cp('-R', path.join(libProjectRoot, 'SalesforceSDK'), appProjectRoot);
```
Comment (bhariharan) [inline tools/postinstall-android.js]: Replaced all ```exec``` calls with the ```shelljs``` equivalent.

**Comment by bhariharan** [comment]
@wmathurin because I'm not sure if you're getting e-mails from this repo.

**Comment by kchitalia** [comment]
:+1: Looks good to me

**Comment by wmathurin** [comment]
:+1:

**Pull request merged**